VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -427,16 +362,13 @@ class VungleAdapter : PartnerAdapter {
     override suspend fun invalidate(partnerAd: PartnerAd): Result<PartnerAd> {
         PartnerLogController.log(INVALIDATE_STARTED)
 
-        listeners.remove(partnerAd.request.identifier)
-        adms.remove(partnerAd.request.partnerPlacement)
-
         return when (partnerAd.request.format.key) {
             /**
              * Only invalidate banner ads.
              * For fullscreen ads, since Vungle does not provide an ad in the load callback, we don't
              * have an ad in PartnerAd to invalidate.
              */
-            AdFormat.BANNER.key, "adaptive_banner" -> destroyBannerAd(partnerAd)
+            AdFormat.BANNER.key, "adaptive_banner" -> finishVungleAd(partnerAd)
```

Comment:
It's fine to keep it as `destroy` since that's more of our lingo. We don't need to name our methods after the partner APIs.

Reply:
Sounds good. 👍🏼